hydra-node/test/Hydra/Model.hs

```diff
@@ -146,15 +153,16 @@ instance StateModel WorldState where
   data Action WorldState a where
     Seed ::
       { seedKeys :: [(SigningKey HydraKey, CardanoSigningKey)]
-      , seedContestationPeriod :: ContestationPeriod
-      , seedDepositDeadline :: DepositDeadline
+      , contestationPeriod :: ContestationPeriod
       , toCommit :: Uncommitted
+      , additionalUTxO :: UTxOType Payment
       } ->
       Action WorldState ()
-    Init :: {party :: Party} -> Action WorldState ()
-    Commit :: {party :: Party, utxoToCommit :: UTxOType Payment} -> Action WorldState ()
-    Decommit :: {party :: Party, decommitTx :: Payment} -> Action WorldState ()
+    Init :: Party -> Action WorldState HeadId
+    Commit :: {headIdVar :: Var HeadId, party :: Party, utxoToCommit :: UTxOType Payment} -> Action WorldState ()
```

Comment:
what is this about?

Reply:
It may not be needed anymore. But this symbolic var captures the actual `HeadId` as observed when doing the real `Init`. I introduced it to not need to pick apart `MockNode` when doing `performCommit` or `performDeposit`. However, the actual `HeadId` used is always `testHeadId` because this code is used (I think): https://github.com/cardano-scaling/hydra/blob/a1959f7986932633bd0f2b80eb37a1ba5bbd893f/hydra-node/test/Hydra/BehaviorSpec.hs#L1159